src/core/scrollcomponent/web/ScrollViewer.js

```diff
@@ -143,11 +150,15 @@ export default class ScrollViewer extends React.Component {
     _windowOnScroll() {
         if (this.props.onScroll) {
             if (this.props.horizontal) {
-                this.scrollEvent.offsetY = 0;
-                this.scrollEvent.offsetX = window.scrollX - this.props.distanceFromWindow;
+                if(this.scrollEvent && this.scrollEvent.nativeEvent && this.scrollEvent.nativeEvent.contentOffset) {
```

Comment:
No need of null checks here, the reason NPE would've happened here was that the complete object was not declared above